projects/ccf-rui/src/app/modules/right-sidebar/right-sidebar.component.html

```diff
@@ -2,7 +2,7 @@
   
   <div class="container">
     <div class='info-button-container'>
-      <ccf-info-button [videoID]="'142hGer4xvU'" [infoTitle]="'HuBMAP CCF Registration User Interface'"></ccf-info-button>
+      <ccf-info-button [videoID]="'ftmAn_XhafY'" [infoTitle]="'HuBMAP CCF Registration User Interface'"></ccf-info-button>
```

Comment:
Can you update to better structure for these? ie <ccf-info-button videoId="ftmAn_blahblah"...